include/psp2kern/kernel/modulemgr.h

```diff
@@ -201,18 +200,307 @@ SceUID ksceKernelGetProcessMainModule(SceUID pid);
 int ksceKernelGetModuleList2(SceUID pid, SceKernelModuleListInfo *infolists, size_t *num);
 
 /**
+ * @brief Get module info
+ *
  * @param[in]  pid   - target pid
- * @param[in]  modid - target modid
+ * @param[in]  modid - target module id
+ * @param[out] info  - info output pointer
+ *
+ * @return 0 on success, < 0 on error.
+ */
+int ksceKernelGetModuleInfo(SceUID pid, SceUID modid, SceKernelModuleInfo *info);
+
+/**
+ * @brief Get module info2
+ *
+ * @param[in]  pid   - target pid
+ * @param[in]  modid - target module id
  * @param[out] info  - info output
  *
  * @return 0 on success, < 0 on error.
  */
 int ksceKernelGetModuleInfo2(SceUID pid, SceUID modid, SceKernelModuleInfo2 *info);
 
+/**
+ * @brief Get module info mini by module address
+ *
+ * @param[in]  pid         - target pid
+ * @param[in]  module_addr - target module address
+ * @param[out] a3          - unknown output
+ * @param[out] a4          - unknown output
+ * @param[out] module_name - module name
+ *
+ * @return 0 on success, < 0 on error.
+ */
+int ksceKernelGetModuleInfoMinByAddr(SceUID pid, const void *module_addr, int *a3, int *a4, SceKernelModuleName *module_name);
```

Comment:
how about changing the name to `ksceKerenlGetModuleNameByAddr` if this only contained the `name`?
or how about giving another struct name like `SceKernelModuleInfo3` :)?

Reply:
need to test ksceKernelGetModuleInfoMinByAddr